src/main/resources/js/condec.dialog.js

```diff
@@ -91,13 +97,18 @@
 		header.textContent = headerText;
 	}
 
-	function setUpCreateOrEditDialog(summary, description, knowledgeType) {
+	function setUpCreateOrEditDialog(summary, description, knowledgeType,addDocumentLocation) {
```

Comment:
We should get rid of the boolean addDocumentationLocation parameter. But therefore we need converting methods, e.g. from JIRA issues to comment.